### DISCUSSION TITLE
Milvus 2.0

### DISCUSSION THREAD

**Discussion by keotic** [milvus-io/milvus#5108] [Q&A and General discussion]
Hi
Is there and estimated ETA for Milvus version 2.0?
Would it be available to download and install on-premises or only cloud based?
Many cool features like DSL query language has gone in version 1.0  and are expected on version 2.0
Thanks

**Comment by hadim** [comment]
We are also very interested in using Milvus 2.0 on-premises. The string ID feature is currently a bit blocking for us since we would like to avoid relying on an additional layer to convert `string_id <-> interger_id`.

**Comment by dengc367** [comment]
I'm also interested in the Estimated Time Of Release of Milvus 2.0? @JinHai-CN

**Reply by yhmo** [reply]
> I'm also interested in the Estimated Time Of Release of Milvus 2.0? @JinHai-CN

JInHai has left the company three months ago for personal reason, he makes lot of contribution to Milvus. For further questions, you can @ me or @shengjun1985

**Comment by yhmo** [comment]
The 2.0 source code is in Master branch, it is still in testing/stable stage, please stay tuned. So far as I know, it required 1 ~ 2 months to reach beta criteria.
